modules/ROOT/pages/index.adoc

```diff
@@ -82,6 +82,110 @@ Monitor APIs and endpoints at scheduled intervals. Receive reports about whether
 
 See xref:api-functional-monitoring::afm-in-anypoint-platform.adoc[API Functional Monitoring in Anypoint Monitoring].
 
+== Anypoint Monitoring Features
+
+Anypoint Monitoring is the standard method of monitoring Mule application performance, logging, and troubleshooting to reduce the mean time to identify and resolve issues. While all customers receive fundamental monitoring capabilities with their subscription, the Titanium tier offers greater storage, guaranteed resources, and advanced capabilities that allow for greater control and flexibility.
+
+|===
+| Feature | Gold/Platinum subscription features | Titanium subscription features
+
+3+| *Built-in Application Dashboards*
+| More than 80 out-of-the-box metrics for Overview, Inbound, Outbound, Performance, Failures, JVM, and Infrastructure charts
+| Yes
+| Yes
+
+| Advanced metrics for Overview, Inbound, Outbound, Performance, Failures, JVM, and Infrastructure charts
+| No
+| Yes
+
+| Flows charts
+| No
+| Yes
+
+| Connectors charts
+| No
+| Yes
+
+| Application Network integration
+| No
+| Yes
+
+| Custom metrics
+| No
+| Yes
+
+| xref:monitoring::performance-and-impact.adoc#data-retention-and-aggregation[Data retention] for more than 30 days
+| No
+| Yes
+
+3+| *Built-in API Dashboards*
+
+| Overview charts
+| Yes
+| Yes
+
+| Raw events
+| No
+| Yes
+
+| xref:monitoring::performance-and-impact.adoc#data-retention-and-aggregation[Data retention] for more than 30 days
+| No
+| Yes
+
+3+| *Custom Dashboards*
+
+| Basic mode
+| Yes
+| Yes
+
+| Advanced mode
+| Yes
+| Yes
+
+| Advanced mode Alert tab
+| No
+| Yes
+
+3+| *Alerts*
+
+| Basic alerts
+| Yes
+| Yes, with xref:performance-and-impact.adoc#data-retention-and-aggregation[higher limits]
+
+| Advanced alerts
+| No
+| Yes
+
+3+| *Logs*
+
+| Log search
+| No
+| Yes
+
+| Log points
+| No
+| Yes
+
+| Raw data
+| No
+| Yes
+
+3+| *Additional Anypoint Monitoring features*
+
+| Anypoint Monitoring Archive API
+| No
+| Yes
+
+| Functional Monitoring
+| Yes
+| Yes
+
+| Reports
+| No
+| Yes
+
+|===
+
```

Comment:
@sarathecoubian A major part of Titanium is dedicated infrastructure. Let's add something like `Dedicated infrastructure for data processing and storage'. It's a big selling point for Titnaium.

Reply:
Added.